scripts/nayduck.py

```diff
@@ -205,17 +207,17 @@ def run_locally(tests):
                 '--',
                 '--exact',
                 fields[3])
-            cwd = None
+            cwd = os.path.join(REPO_DIR, 'pytest')
         elif fields[0] in ('pytest', 'mocknet'):
             fields[0] = sys.executable
-            fields[1] = os.path.join('tests', fields[1])
+            fields[1] = os.path.join('./pytest/tests', fields[1])
             cmd = fields
-            cwd = 'pytest'
+            cwd = os.path.join(REPO_DIR, 'pytest')
```

Comment:
I would leave `fields[1]` alone and than `cwd = REPO_DIR / 'pytest'`. As it stands pytests won’t work.

Reply:
Thanks, fixed.